### PR TITLE
Cu 8696hrm29 create profile page patryk kosiski

### PR DESCRIPTION
**Pull request by JimTheCat**

### CODE REVIEW

**Comment by JimTheCat** [comment]
Task linked: [CU-8696hrm29 Create profile page](https://app.clickup.com/t/8696hrm29)

**Comment by KinTrae** [comment]
fix your builds first

File: frontend/src/Components/Cards/ProfileFriends/ProfileFriends.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  const navigate = useNavigate();
+
+  return (
+    <Card shadow="sm" px="lg" pt={"lg"} radius="md" w={400} withBorder>
+      <Group justify={"space-between"} mb={15}>
+        <Text size={"lg"}>Friends</Text>
+        <UnstyledButton component="a" href="https://www.google.com" onClick={() => navigate("https://www.google.com")}
+                        c="dimmed">Wyświetl
+          wszystko</UnstyledButton>
```
Comment (KinTrae) [inline frontend/src/Components/Cards/ProfileFriends/ProfileFriends.tsx:15]: i don't like this enter

File: frontend/src/Components/Cards/ProfileMultimedia/ProfileMultimedia.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+export const ProfileMultimedia = (props: { multimedia: Multimedia[] }) => {
+
+  const navigate = useNavigate();
+
+  return (
+    <Card shadow="sm" px="lg" pt={"lg"} radius="md" w={400} withBorder>
+      <Group justify={"space-between"} mb={15}>
+        <Text size={"lg"}>Multimedia</Text>
+        <UnstyledButton onClick={() => navigate("https://www.google.com")} c="dimmed">Wyświetl
+          wszystko</UnstyledButton>
```
Comment (KinTrae) [inline frontend/src/Components/Cards/ProfileMultimedia/ProfileMultimedia.tsx:14]: i don't like this enter

File: frontend/src/Components/InnerHtmlHandler/InnerHtmlHandler.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import {TypographyStylesProvider} from "@mantine/core";
+import DOMPurify from 'dompurify';
+
+export const InnerHtmlHandler = (props: { innerHtml: string }) => {
+
+  // const allowedArgs = {
+  //   //TODO: Define allowed tags and attributes
+  //   ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a', 'p', 'ul', 'li', 'br'],
```
Comment (KinTrae) [inline frontend/src/Components/InnerHtmlHandler/InnerHtmlHandler.tsx:8]: why commented code?

**Review by KinTrae — COMMENTED** [review]
Also please check sonar report

**Review by KinTrae — APPROVED** [review]
ok. Goob job:)

**Pull request merged**